include/PruManager.h

```diff
@@ -0,0 +1,78 @@
+/* 
+ * PruManager.h
+ *
+ * includes the classes required for introducing RProc functionality
```

Comment:
a more appropriate description would be "support for interaction with PRU via rproc/mmap and/or uio/libprussdrv"